src/main/java/net/sf/marineapi/nmea/io/UDPDataReader.java

```diff
@@ -1,6 +1,6 @@
 /*
  * UDPDataReader.java
- * Copyright (C) 2010-2014 Kimmo Tuukkanen, Ludovic Drouineau
+ * Copyright (C) 2010-2019 Kimmo Tuukkanen, Ludovic Drouineau

```

Comment:
Only new files should be updated with "2019". The general rule is that copyright notice shows the first year of publishing and it is not updated afterwards.

Reply:
Done